noxfile.py

```diff
@@ -123,10 +130,27 @@ def fix(session: Session) -> None:
         "--fix",
         f"{Settings.VERSION_FILE}",
     )
+    files = [f"{path}" for path in _python_files(PROJECT_ROOT)]
+    session.run(
+        "poetry",
+        "run",
+        "python",
+        "-m",
+        "pyupgrade",
+        "--py38-plus",
+        "--exit-zero-even-if-changed",
+        *files,
+    )
```

Comment:
why do you calling pyuprdae here directly and have further down a session

Reply:
ah, I see it --exit-zero-even-if-changed, so one is for precommit and one for CI or local, maybe explain this a bit or have to session for pyupgrade where the name explains it

Reply:
Isn't it clear enough from the fact that it is part of the "fix" session? 
What would be best in your opinion `comment`?

Reply:
maybe move it also to a session with the name pyupgrade_fix or fix_pyupgrade and call this session.

Reply:
Hi @tkilias, thanks for clarifying, still I personally rather would keep it as is, reasoning:
* overhead is not worth it in my opinion
* consistency with the other call's within the fix action

anyway I will go with whatever your final call will be.

Reply:
Third option, I think, my problem was understanding that this call is actually for configured for fix. Maybe, it is enough to extract it to a method